client.go

```diff
@@ -110,6 +110,10 @@ func (c *Client) Store(req Request, dest string) error {
 	if err != nil {
 		return err
 	}
+	// Check for 2XX Status Codes
+	if resp.StatusCode < http.StatusOK || resp.StatusCode >= http.StatusMultipleChoices {
```

Comment:
Does this function `Client.Store()` only get called for conversion to PDFs?

Reply:
`Client.Store()` only gets called for the conversion to PDFs or for merging several PDF files. I've updated the error message returned to reflect the same.